lib/selector.rb

```diff
@@ -10,7 +12,15 @@ class Selector
 
   def initialize(options, prompt = @@default_prompt, timeout = 10)
     @options = options
-    @timeout = timeout
+    # Set timeout to zero if a non-interactive console.
+    # Check noninteractive usage with `setsid command`.
+    @timeout = if !IO.console&.console_mode || IO.console&.winsize == [0, 0]
+      # crewlog "IO.console&.console_mode is #{IO.console&.console_mode}"
+      # crewlog "IO.console&.winsize is #{IO.console&.winsize}"
```

Comment:
Uncomment?

Reply:
Removed! They were there to help debug a progress_bar issue.